### DISCUSSION TITLE
[Question] Maps World of Warcraft (master = 9.2.7.45745)

### DISCUSSION THREAD

**Discussion by xBarselonx** [TrinityCore/TrinityCore#28569] [Q&A] [answered]
Are there maps available for download (dbc, maps, vmaps, mmaps, cameras and gte) for ruRU version (master = 9.2.7.45745)? For example, AzerothCore does this for World of Warcraft 3.3.5a.

**Comment by Jinnaix** [comment, marked as answer]
No. That would be on the same line as distributing the client itself and that's clearly against copyright. The project you mentioned is trying to avoid legal problems in that case by not hosting them under their official account. We do not provide game data files in any way.

**Reply by xBarselonx** [reply]
Thanks for the clarification.
